frontend/talentsearch/src/js/components/Home/partials/Opportunities/Opportunities.tsx

```diff
@@ -0,0 +1,173 @@
+                  id: "+cBKDC",
+                  description:
+                    "Heading for the digital government job opportunities",
+                }),
+                summary: intl.formatMessage({
+                  defaultMessage:
+                    "Check out the latest GC opportunities in digital and tech, from entry level to management. Find a team, make a difference, and be inspired.",
+                  id: "951Oju",
+                  description:
+                    "Description for the digital government job opportunities",
+                }),
+                link: {
+                  path: diPaths.allPools(),
+                  label: intl.formatMessage({
+                    defaultMessage: "Browse IT jobs",
+                    id: "zNvXSs",
+                    description:
+                      "Link text for IT jobs in government call to action",
+                  }),
+                },
+              }}
+            />
+            <Block
+              content={{
+                color: "blue",
+                title: intl.formatMessage({
+                  defaultMessage: "Indigenous Apprenticeship Program",
+                  id: "XR37x0",
+                  description:
+                    "Heading for the Indigenous Apprenticeship Program on home page",
+                }),
+                summary: intl.formatMessage({
+                  defaultMessage:
+                    "Designed by the Indigenous community for the Indigenous community, this program recruits entry-level applicants for learning and development IT opportunities across government.",
+                  id: "f3Qqop",
+                  description:
+                    "Description for the Indigenous Apprenticeship Program on home page",
+                }),
+                link: {
+                  path: `${locale}/indigenous-it-apprentice`,
+                  label: intl.formatMessage({
+                    defaultMessage:
+                      "Apply<hidden> to the Indigenous Apprenticeship Program</hidden> now",
+                    description:
+                      "Link text to apply for the Indigenous Apprenticeship Program",
+                    id: "w3Kkk2",
+                  }),
+                },
+              }}
+            />
+            <Block
+              content={{
+                color: "red",
+                title: intl.formatMessage({
+                  defaultMessage: "Executives in digital government",
+                  id: "9KOwXq",
+                  description: "Heading for executive jobs in government",
+                }),
+                summary: intl.formatMessage({
+                  defaultMessage:
+                    "From entry-level executives to CIO opportunities across the GC, this is the place to come if you’re ready to take on a digital leadership role making a difference for Canadians.",
+                  id: "pDGY5P",
```

Comment:
I think we're trying to avoid using fancy unicode characters when possible.  Also:
https://github.com/GCTC-NTGC/gc-digital-talent/blob/8e42a6967b5197b6a62ae05e34a677be5fec1fec/frontend/talentsearch/src/js/components/Home/partials/Hero/Hero.tsx#L67 
https://github.com/GCTC-NTGC/gc-digital-talent/blob/8e42a6967b5197b6a62ae05e34a677be5fec1fec/frontend/talentsearch/src/js/components/Home/partials/Profile/Profile.tsx#L78

Reply:
Thanks! I wasn't sure of our position on this. I have removed them with [remove special characters](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4049/commits/b5c4575e99eda624868438e20f2ec168e2646d0f)

Reply:
I like the simplicity of using straight characters, but it seems we may be "doing it wrong" :/ (at least according to a few opinionated guides like this one https://smartquotesforsmartpeople.com/ )

I'll spin off a non-urgent issue to discuss.

Related:
- https://github.com/totallymoney/eslint-plugin-prefer-smart-quotes
- https://github.com/seleb/eslint-plugin-no-smart-quotes
- https://medium.com/kabbage/update-to-smart-quotes-with-one-line-of-code-987152a9342b
- https://www.npmjs.com/package/eslint-plugin-json (maybe? for arbitrary eslint rules on fr.json)